bin/exec-env

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+if [ -n "$ASDF_INSTALL_PATH" ]; then
```

Comment:
Please also test that JAVA_HOME is unset, before exporting it.

Reply:
Wouldn't that defeat the purpose of the change?

`JAVA_HOME` should always point to the currently used java installation.

eg.
```
export JAVA_HOME=/some/java/install/bin
...
asdf local java adoptopenjdk-8.0.292+10
asdf env java echo $JAVA_HOME
/Users/foo/.asdf/installs/java/adoptopenjdk-8.0.292+10/bin/java
```

Reply:
right, then for me it's good to merge.